tests/unit/test-util/test.adb.js

```diff
@@ -622,6 +623,75 @@ describe('utils/adb', () => {
        });
   });
 
+  describe('checkOrCleanArtifacts', () => {
+    const artifactDir = `web-ext-artifacts-${Date.now()}`;
+    function makeFakeArtifact(artifactName, isDirectory) {
+      return {
+        name: artifactName,
+        isDirectory: () => {
+          return isDirectory;
+        },
+      };
+    }
+    const files = [
+      makeFakeArtifact(artifactDir.concat('fake1'), true),
+      makeFakeArtifact(artifactDir.concat('fake2'), true),
+      makeFakeArtifact(artifactDir.concat('fake2'), false),
+      makeFakeArtifact('not-web-ext-artifacts', true),
```

Comment:
nit, lets rename these fake files as `fake-dir1`, `fake-dir2`, `fake-file3`, and `not-web-ext-artifacts`
(fake2 is currently repeated twice, also fake1 and fake2 doesn't tell us much, making clear the role of that fake file by including `dir` and `file` in their name would make it almost immediately clear).